server/sonar-db-dao/src/main/java/org/sonar/db/organization/OrganizationMemberDao.java

```diff
@@ -55,4 +58,21 @@ public void deleteByOrganizationUuid(DbSession dbSession, String organizationMem
     return mapper(dbSession).selectOrganizationUuidsByUser(userId);
   }
 
+  /**
+   *
+   * @param loginOrganizationConsumer {@link BiConsumer}<String,String> (login, organization uuid)
+   */
+  public void selectForUserIndexing(DbSession dbSession, List<String> logins, BiConsumer<String, String> loginOrganizationConsumer) {
+    executeLargeInputsWithoutOutput(logins, list -> mapper(dbSession).selectForIndexing(list)
+      .forEach(row -> loginOrganizationConsumer.accept(row.get("login"), row.get("organizationUuid"))));
```

Comment:
I'm just a little worried about the mix usage of user id and user login in this DAO, but I understand than using user id in the 2 new classes would add complexity into UserResultSetIterator.
In fact we should replace user_id by login in tables, but this is out of scope of this PR.